jobdispatcher/src/testLib/com/firebase/jobdispatcher/TestUtil.java

```diff
@@ -87,6 +87,15 @@ public static void assertHasSinglePrivateConstructor(Class<?> cls) throws Except
     constructor.newInstance();
   }
 
+  public static void assertHasSinglePrivateUnusableConstructor(Class<?> cls) {
+    try {
+        assertHasSinglePrivateConstructor(cls);
```

Comment:
any chance we could just use Junit 4.13's assertThrows here?

Reply:
4.13 isn't available yet AFAIK, but I wish.

Reply:
bummer. Ok, sgtm